### DISCUSSION TITLE
Load Power sensor: Is it "AC" outlet only or is it AC + Backup combined?

### DISCUSSION THREAD

**Discussion by smoketabs** [nathanmarlor/foxess_modbus#325] [Q&A] [answered]
Hi.

I have both the AC connection and Backup connection live into my consumer unit. So there's obviously "two" loads.

Regarding the "Load Energy Today", "Load Energy Total" and "Load Power" sensors, are these only reading the "main" AC connection? Or is it a combined AC + Backup connection reading?

I ask, as I want to make sure true consumption is being represented.

If not, then I suppose I need to create some form of sensor wizardry which sums the two together?

This is also pertinent to your EM integration (perhaps even more important as the backup outlet is essentially my "ghost load" (fridges/IT kit/lights/smoke alarms).

Thanks.
(I'll also post this to the EM discussion)

**Comment by canton7** [comment]
I'm not actually sure to be honest. Can you do a quick test to find out? E.g. turn on/off a load on the EPS, see whether the load value goes up/down

**Reply by smoketabs** [reply, marked as answer]
You know what, I'm one of those users I used to hate!!! I knew there was a reason why I put some Eastrons at the front of each of the circuit groupings (Non-EPS & EPS)!!!

Looks like the "Load Power" sensor is total load leaving the inverter (AC + Backup):

![Screenshot 2023-06-09 at 15 24 34](https://github.com/nathanmarlor/foxess_modbus/assets/15179147/a0c8ccdf-8185-4f60-9742-7d75e0719162)

**Reply by canton7** [reply]
Lovely, thanks for that!
